process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -295,6 +296,19 @@ func (st *storageBootstrapper) applyHeaderInfo(hdrInfo bootstrapStorage.Bootstra
 	return nil
 }
 
+func (st *storageBootstrapper) getScheduledRootHash(headerFromStorage data.HeaderHandler, headerHash []byte) []byte {
+	rootHash := headerFromStorage.GetRootHash()
+
+	scheduledRootHash, err := st.scheduledTxsExecutionHandler.GetScheduledRootHashForHeader(headerHash)
+	if err == nil {
```

Comment:
a log.Warn in case of error ? This is old code, but still.

Reply:
This fi executes when there is not an error -> err == nil